apps/ledger-live-desktop/tests/page/portfolio.page.ts

```diff
@@ -1,11 +1,19 @@
 import { step } from "tests/misc/reporters/step";
 import { AppPage } from "tests/page/abstractClasses";
+import { expect } from "@playwright/test";
 
 export class PortfolioPage extends AppPage {
   readonly emptyStateTitle = this.page.getByTestId("portfolio-empty-state-title");
   private addAccountButton = this.page.getByTestId("portfolio-empty-state-add-account-button");
   private buySellEntryButton = this.page.getByTestId("buy-sell-entry-button");
+  private swapEntryButton = this.page.getByTestId("swap-entry-button");
   private stakeEntryButton = this.page.getByTestId("stake-entry-button");
+  private chart = this.page.getByTestId("chart-container");
+  private marketPerformanceWidget = this.page.getByTestId("market-performance-widget");
+  private swapButton = this.marketPerformanceWidget.locator('button:has-text("Swap")');
+  private buyButton = this.marketPerformanceWidget.locator('button:has-text("Buy")');
```

Comment:
Maybe we can use `getByRole` instead of `locator`

Reply:
good point! done :)